web-client/app/styles/main.scss

```diff
@@ -1153,6 +1153,7 @@ ul.cards-list-container {
         margin: 10px;
         display: block;
         padding-top: 1em;
+        overflow: scroll;
```

Comment:
You might also need to add: `word-wrap: initial;` but not sure. I tried to edit CSS on the fly when the highlight.js failed to kick in and using overflow by itself didn't seem to work. Have you tried this?